app/controllers/media_controller.rb

```diff
@@ -1,5 +1,5 @@
 class MediaController < BaseMediaController
-  before_filter :authenticate_if_private
+  before_action :authenticate_if_private
```

Comment:
I think it would be worth explaining in the commit note that [`before_action` is just a renaming of `before_filter`](https://github.com/rails/rails/commit/9d62e04838f01f5589fa50b0baa480d60c815e2c), i.e. there is no inherent change in behaviour.